lib/stream/dev_bundle.js

```diff
@@ -0,0 +1,95 @@
+var _ = require("lodash");
+var through = require("through2");
+var hasES6 = require("../graph/has_es6");
+var unbundle = require("../graph/unbundle");
+var flattenBundle = require("../bundle/flatten");
+var makeBundle = require("../bundle/make_bundle");
+var splitByBuildType = require("../bundle/split_by_build_type");
+var addTraceurRuntime = require("../bundle/add_traceur_runtime");
+var makeBundlesConfig = require("../bundle/make_bundles_config");
+
+var BUNDLE_DEPTH = 1;
+
+module.exports = function() {
+	return through.obj(function(data, enc, done) {
+		try {
+			var result = bundle(data);
+			done(null, result);
+		} catch(err) {
+			done(err);
+		}
+	});
+};
+
+function bundle(data) {
+	var configGraph = data.configGraph;
+	var configuration = data.configuration;
+	var graph = includePluginsInBuild(data.graph);
+
+	unbundle(graph);
+
+	var mainBundle = makeBundle(graph);
+	flattenBundle(mainBundle, BUNDLE_DEPTH);
+	var splitMainBundles = nameBundles(splitByBuildType(mainBundle));
+
+	splitMainBundles.filter(isJSBundle).forEach(function(bundle) {
+		var unshift = [].unshift;
+		unshift.apply(bundle.nodes, configGraph);
+
+		// Make config JS code so System knows where to look for bundles.
+		var configNode = makeBundlesConfig(
+			splitMainBundles,
+			configuration,
+			bundle,
+			{ excludedBundles: {} }
```

Comment:
I'd have to pass an empty object because of https://github.com/stealjs/steal-tools/blob/master/lib/bundle/make_bundles_config.js#L12.... I'll change the module to make `options` optional and get rid of this line,